src/mappings/doid.sssom.tsv

```diff
@@ -275,13 +273,10 @@ DOID:0050427	xeroderma pigmentosum	oboInOwl:hasDbXref	MESH:D014983	semapv:Unspec
 DOID:0050427	xeroderma pigmentosum	oboInOwl:hasDbXref	NCI:C3452	semapv:UnspecifiedMatching
 DOID:0050427	xeroderma pigmentosum	oboInOwl:hasDbXref	ORDO:910	semapv:UnspecifiedMatching
 DOID:0050427	xeroderma pigmentosum	oboInOwl:hasDbXref	UMLS:C0043346	semapv:UnspecifiedMatching
-DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	MESH:D015776	semapv:UnspecifiedMatching
-DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	NCI:C3147	semapv:UnspecifiedMatching
-DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	OMIM:600962	semapv:UnspecifiedMatching
-DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	ORDO:2337	semapv:UnspecifiedMatching
-DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	UMLS:C0022584	semapv:UnspecifiedMatching
-DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	UMLS:C3489771	semapv:UnspecifiedMatching
-DOID:0050428	nonepidermolytic palmoplantar keratoderma	skos:exactMatch	OMIM:600962	semapv:UnspecifiedMatching
+DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	MESH:C563422	semapv:UnspecifiedMatching
+DOID:0050428	nonepidermolytic palmoplantar keratoderma	oboInOwl:hasDbXref	UMLS:C1833030	semapv:UnspecifiedMatching
+DOID:0050428	nonepidermolytic palmoplantar keratoderma	skos:exactMatch	MESH:C563422	semapv:UnspecifiedMatching
+DOID:0050428	nonepidermolytic palmoplantar keratoderma	skos:exactMatch	UMLS:C1833030	semapv:UnspecifiedMatching
```

Comment:
What does `skos:exactMatch` mean in the context of this sssom file and why are there now `oboInOwl:hasDbXref` and `skos:exactMatch`?

Reply:
Hmm, that's a good question. I'll have to look into that. I believe these files are from the lexmatch pipeline that Harshad wrote. I don't know if it is a proposed match, or a match that is actually declared in the source.

I'm not sure why`hasDbXref` is in here either, but I looked and my development environment also has these, so this predates this PR.

Reply:
When you have questions / hypotheses like this its good to consult the `Makefile` first:

https://github.com/monarch-initiative/mondo-ingest/blob/52ec97cfe3d64c5aa49f49393fe2735c2a45ca78/src/ontology/mondo-ingest.Makefile#L187

These `src/mapping/` files have nothing to do with `lexmatch`, they are just 1:1 extracts from the components (in this case DOID). So whatever DO says, will appear in this file.